conda/environments/cuml_dev_cuda11.0.yml

```diff
@@ -23,6 +23,10 @@ dependencies:
 - umap-learn
 - scikit-learn=0.23.1
 - treelite=1.3.0
+- statsmodels
```

Comment:
What are these new dependencies for? Just curious.

Reply:
They're required in cuML Python tests, but only listed in the `rapids-test-env` metapackage.